ddl/ddl_api.go

```diff
@@ -1690,9 +1690,7 @@ func checkConstraintNames(constraints []*ast.Constraint) error {
 
 	// Set empty constraint names.
 	for _, constr := range constraints {
-		if constr.Tp == ast.ConstraintForeignKey {
-			setEmptyConstraintName(fkNames, constr, true)
```

Comment:
Please also update `setEmptyConstraintName` since the third arg is a non-variant.